enterprise_access/apps/subsidy_request/models.py

```diff
@@ -0,0 +1,162 @@
+""" Models for subsidy_request. """
+
+from uuid import uuid4
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+from enterprise_access.apps.subsidy_request.constants import SubsidyRequestStates
+
+
+class SubsidyRequest(TimeStampedModel):
+    """
+    Stores information related to a request for a subsidy (license or coupon).
+
+    .. no_pii: This model has no PII
+    """
+
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid4,
+        editable=False,
+        unique=True,
+    )
+
+    lms_user_id = models.IntegerField()
+
+    course_id = models.CharField(
+        null=True,
+        blank=True,
+        max_length=128
+    )
+
+    customer_uuid = models.UUIDField()
```

Comment:
Nit: we call this `enterprise_customer_uuid` in license-manager and enterprise-customer, should we use the same name here?

Reply:
yep we should do that!